src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -88,26 +61,31 @@ public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossT
     return tenantsToQuery;
   }
 
-  public String getCentralTenantId() {
+  @SuppressWarnings("unchecked") // JsonPath.parse is returning a plain List without a type parameter, and the TypeRef (vs Class) parameter to JsonPath.read is not supported by the JSON parser
+  private List<Map<String, String>> getUserTenants() {
     try {
-      String rawJson = ecsClient.get(CONSORTIA_CONFIGURATION_PATH, Map.of());
-      DocumentContext parsedJson = JsonPath.parse(rawJson);
-      return parsedJson.read(CENTRAL_TENANT_ID);
+      String userTenantsResponse = ecsClient.get("user-tenants", Map.of());
```

Comment:
```suggestion
      String userTenantsResponse = ecsClient.get("user-tenants", Map.of("limit", String.valueOf(1000)));
```

I think this should resolve the problem pointed out in https://folio-org.atlassian.net/browse/MODFQMMGR-465 (at least up to 1000 tenants)